flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/PartitionMonitorTest.java

```diff
@@ -99,6 +99,10 @@ private void commitPartitionWithGivenCreateTime(
     private void preparePartitionMonitor() {
         List<List<String>> seenPartitionsSinceOffset = new ArrayList<>();
         JobConf jobConf = new JobConf();
```

Comment:
do we have nay more test to verify the changes, e.g. the split number will change with `table.exec.hive.file-open-cost` change and `table.exec.hive.split-max-size` change

Reply:
Now, I have added a test in [HiveSourceFileEnumeratorTest.java](https://github.com/apache/flink/pull/20377/files#diff-059eba29b7890b2ab4886612e37be7ca35f255ce8519a5afb43d1a3a9f34b401)  for it.

Reply:
Also, when adding test, I found the current implementation only works for orc format. When it's other format, although we set `mapreduce.input.fileinputformat.split.maxsize`, but the other formats won't take this congfiguration as consideration when call method `InputFormat#getSplits` to get splits.

So, I add a note in the doc to tell these related configuration only works for orc format. And  I changes the code so that we only try to set `mapreduce.input.fileinputformat.split.maxsize` when it's orc format.
As a result, the pr titlte also changes.